src/027.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  description: 'Single Value with bool type and a single Point feature.',
+  specification_reference: 'https://github.com/mapbox/vector-tile-spec/blob/master/2.1/README.md',
+  validity: {
+    v1: true,
+    v2: true,
+    error: 'fatal'
```

Comment:
Since this is a valid fixture, we don't need an `error` field @GretaCB - I'll remove it!